src/opm/output/eclipse/AggregateConnectionData.cpp

```diff
@@ -71,6 +71,8 @@ namespace {
 	for (const auto & conn : conns) {
 	    std::size_t sI = conn.getCompSegSeqIndex();
 	    cs_seqIndConnMap.insert(std::make_pair(sI, &conn));
+	    std::cout << "mapCSSITC: connSI sI: " << sI <<  " I: " << conn.getI() << " J: " << conn.getJ()
```

Comment:
This looks like a debug message?

Reply:
I think all the debug statements are removed in the final version of the code!?

Reply:
Yes you are right - I looked at the first commit.